esrally/mechanic/telemetry.py

```diff
@@ -276,6 +276,159 @@ def detach_from_node(self, node, running):
             self.attached = False
 
 
+class CCRStats(TelemetryDevice):
+    internal = False
+    command = "ccr-stats"
+    human_name = "CCR Stats"
+    help = "Regularly samples CCR related stats"
+
+    """
+    Gathers CCR stats on a cluster level
+    """
+
+    def __init__(self, telemetry_params, clients, metrics_store):
+        """
+        :param telemetry_params: The configuration object for telemetry_params.
+            May optionally specify:
+            ``ccr-stats-indices``: JSON object specifying the indices per cluster to pushish statistics from.
```

Comment:
This was meant to refer to https://www.w3schools.com/js/js_json_objects.asp and not the specific data type because we support reading both a JSON string directly in the cli arg or from a JSON filename.

Reply:
Ok